v1/pipeline/stages/cd/deployment-stage-config.yaml

```diff
@@ -1,35 +1,46 @@
+$schema: http://json-schema.org/draft-07/schema#
 title: DeploymentStageConfig
 type: object
 required:
-- execution
+- steps
 properties:
-  customDeploymentRef:
+  custom_deployment_ref:
     $ref: step-template-ref.yaml
-  deploymentType:
+  deployment:
     type: string
     enum:
-    - Kubernetes
-    - NativeHelm
-    - Ssh
-    - WinRm
-    - ServerlessAwsLambda
-    - AzureWebApp
-    - CustomDeployment
-    - ECS
-    - Elastigroup
-    - TAS
-    - Asg
-    - GoogleCloudFunctions
-    - AwsLambda
-    - AWS_SAM
+    - kubernetes
+    - native-helm
+    - ssh
+    - win-rm
+    - serverless-aws-lambda
```

Comment:
can we simplify this

Reply:
Not really sure how. This is minimal naming possible while keeping intuitive